packages/react-component-library/src/components/RadioCard/RadioCard.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useRef } from 'react'
+
+import { Radio, RadioProps } from '../Radio'
+
+export interface RadioCardProps extends Omit<RadioProps, 'label'> {
+  title: string
+  description?: string
+  tabIndex?: number
+}
+
+export const RadioCard: React.FC<RadioCardProps> = ({
+  name,
+  title,
+  description,
+  tabIndex = 0,
+  isChecked,
+  ...rest
+}) => {
+  const ref = useRef(null)
+
+  const handleClick = (_: React.MouseEvent<HTMLDivElement>) => {
+    ref.current.click()
+  }
+
+  const handleKeyUp = (_: React.KeyboardEvent) => {
+    ref.current.focus()
+  }
+
+  return (
+    <div
```

Comment:
I have been trying to keep the attributes alphabetical. Seems to be easier to read.